packages/enzyme/src/index.js

```diff
@@ -15,4 +16,6 @@ module.exports = {
   ReactWrapper,
   configure,
   EnzymeAdapter,
+  trackMountedWrapper,
```

Comment:
why does this need to be re-exported?

Reply:
for custom implementation of `mount` that extend the enzyme wrapper. We use this pattern often, for example, to encapsulate logic around application context.

Reply:
I don't think it needs to be in the root export tho - consumers can deep-import whatever they need from inside `build`